File: Admin_p/Admin_p.Web/Modules/ServerTypes/electroDB/PaymentsForm.ts
import { ServiceLookupEditor, StringEditor, DateEditor, DecimalEditor, PrefixedContext, initFormType } from "@serenity-is/corelib";

export interface PaymentsForm {
    OrderId: ServiceLookupEditor;
    PaymentMethod: StringEditor;
    PaymentStatus: StringEditor;
    TransactionDate: DateEditor;
    Amount: DecimalEditor;
    CreatedAt: DateEditor;
    UpdatedAt: DateEditor;
    TransactionId: StringEditor;
}

export class PaymentsForm extends PrefixedContext {
    static readonly formKey = 'electroDB.Payments';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!PaymentsForm.init)  {
            PaymentsForm.init = true;

            var w0 = ServiceLookupEditor;
            var w1 = StringEditor;
            var w2 = DateEditor;
            var w3 = DecimalEditor;

            initFormType(PaymentsForm, [
                'OrderId', w0,
                'PaymentMethod', w1,
                'PaymentStatus', w1,
                'TransactionDate', w2,
                'Amount', w3,
                'CreatedAt', w2,
                'UpdatedAt', w2,
                'TransactionId', w1
            ]);
        }
    }
}

File: Admin_p/Admin_p.Web/Modules/electroDB/Payments/PaymentsForm.cs
using Serenity.ComponentModel;
using System;

namespace Admin_p.electroDB.Forms;

[FormScript("electroDB.Payments")]
[BasedOnRow(typeof(PaymentsRow), CheckNames = true)]
public class PaymentsForm
{
    public int OrderId { get; set; }
    public string PaymentMethod { get; set; }
    public string PaymentStatus { get; set; }
    public DateTime TransactionDate { get; set; }
    public decimal Amount { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string TransactionId { get; set; }
}